MonoGame.Framework/Graphics/GraphicsDevice.OpenGL.cs

```diff
@@ -118,32 +122,43 @@ private void PlatformSetup()
                     0;
 
                 var samples = 0;
-                if (Game.Instance.graphicsDeviceManager.PreferMultiSampling)
+                if (Game.Instance != null)
                 {
-                    // Use a default of 4x samples if PreferMultiSampling is enabled
-                    // without explicitly setting the desired MultiSampleCount.
-                    if (PresentationParameters.MultiSampleCount == 0)
+                    if (Game.Instance.graphicsDeviceManager.PreferMultiSampling)
```

Comment:
Do we really need this?  Isn't `PresentationParameters.MultiSampleCount == 0` essentially mean we don't want multisample?